README.md

```diff
@@ -1 +1,109 @@
-# domain-primitives-java
\ No newline at end of file
+# Domain Primitives Java
+
+![License](https://img.shields.io/hexpm/l/plug)
+
+Primitive, self-validating immutable object types.
+
+*Invented by [Stefan Ludwig](https://github.com/slu-it)*
+
+# Table of Contents
+
+* 🚀 [Features](#features)
+* 📚 [Releases](#releases)
+* 👩‍💻/👨‍💻 [Contributing](#contributing)
+* 📨 [Contact](#contact)
+
+# 🚀Features
+
+## Primitive Type
+
+The `PrimitiveType<T>` could be used to create a self-validating immutable object:
+
+```java
+class Name extends PrimitiveType<String> {
+    public Name(String value) {
+        super(value, "Name",
```

Comment:
Vllt eine Überlegung wert, ob wir als Default nicht ein getClass().getSimpleName() für den Namen definieren

Reply:
Ist bereits so: https://github.com/domain-primitives/domain-primitives-java/blob/68aa19bf0d30df0c9240dc2a12f5bef02d83929f/src/main/java/de/novatec/domainprimitives/type/PrimitiveType.java#L13 🙂 

Wollte hier nur das Label mit Absicht mitgeben, um ein klares Beispiel zu haben. Wenn es aber nur verwirrt, nutzen wir hier den default.

Reply:
Update: Können wir gerne mache, das würde ich aber in einer expliziten Doku unterbringen. In der Readme wollte ich nur ein "schnelles" Beispiel abbilden 🙂 
Doku würde ich aber prinzipiell gerne vermeiden, hab versucht solche Defaults (hoffentlich gut) über die Java Docs zu erklären.

Andere Punkt: Ich denke wir sollte in Englisch kommunizieren oder?

Reply:
True ;)

If it is ja simple example then no one can really deduce why the "Name" String is there.
Therefore i wouldn't have the label in the most primitve example and would make a dedicated example for labeling.